app/controllers/followers_controller.rb

```diff
@@ -2,13 +2,10 @@ class FollowersController < ApplicationController
 
   # get a followers list for a particle user
   def followers_list
-    ids = User.find_by_id(params[:id]).followers.map { |i| i.follower_id }
-    @users = User.where(id: ids)
+    @users = User.find_by_id(params[:id]).followers
```

Comment:
User.followers and User.followings return records from Followers table, not Users table. So, I have created a model method that will return the followers and followings.

`def self.followers_list(user_id)
    User.joins(:followers).where(followers: { following_id: user_id })
  end

  def self.followings_list(user_id)
    User.joins(:followers).where(followers: { follower_id: user_id })
  end`
  
  What do you think?